development/build/scripts.js

```diff
@@ -9,11 +8,9 @@ const watchify = require('watchify')
 const browserify = require('browserify')
 const envify = require('envify/custom')
 const sourcemaps = require('gulp-sourcemaps')
-const sesify = require('sesify')
 const terser = require('gulp-terser-js')
 const pify = require('pify')
 const endOfStream = pify(require('end-of-stream'))
-const { makeStringTransform } = require('browserify-transform-tools')
```

Comment:
Could this package be removed from the manifest as well? It seems to be no longer used

Reply:
should be a lint issue, right?

Reply:
I don't think we have a linter that checks for unused dependencies, but we should!